scripts/LangFixes.zs

```diff
@@ -74,6 +74,18 @@ game.setLocalization("en_US", "entity.ItemFrame.name", "Item Frame");
 
 game.setLocalization("en_US", "tile.warpdrive.detection.siren32767.name", "Air Raid Siren");
 
+game.setLocalization("en_US", "enchantment.Soul Tether I", "Soul Tether I");
+
+game.setLocalization("en_US", "death.attack.Overheat", "was killed by overheating their power armor.");
```

Comment:
Too literal. It needs to be something we can laugh about like:
"was baked in their power armor."

Reply:
Updated in https://github.com/DrParadox7/Lost-Era-Modpack/pull/36/commits/e203d2614dda8fdb00520a87c1a48590077fea99